entity-types/infra-awscodebuild/dashboard.json

```diff
@@ -0,0 +1,335 @@
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "BYTES_PER_SECOND"
+            }
+          }
+        },
+        {
+          "title": "CPU and Memory Utilized in % (Average)",
+          "layout": {
+            "column": 7,
+            "row": 9,
+            "width": 6,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT average(aws.codebuild.CPUUtilizedPercent) as 'Average CPU Utilized', average(aws.codebuild.MemoryUtilizedPercent) as 'Average Memory Utilized' FACET aws.codebuild.ProjectName timeseries auto"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "PERCENTAGE"
+            }
+          }
+        },
+        {
+          "title": "CPU Utilized in CPU units (Average)",
+          "layout": {
+            "column": 1,
+            "row": 12,
+            "width": 6,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT average(aws.codebuild.CPUUtilized) FACET aws.codebuild.ProjectName timeseries auto"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Memory Utilized in Megabytes (Average)",
+          "layout": {
+            "column": 7,
+            "row": 12,
+            "width": 6,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT average(aws.codebuild.MemoryUtilized) FACET aws.codebuild.ProjectName timeseries auto"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
```

Comment:
NIT: You can add the unit as in the others

Reply:
@lujop, the unit for Memory Utilisation is Mega bytes and it is not supported in NewRelic. I've added unsupported units in widget titles now. Please review again. Thanks.